auth-web/src/views/auth/HomeView.vue

```diff
@@ -16,30 +16,43 @@
 
       <header class="hero-banner">
         <v-container>
-          <h1>Welcome to Cooperatives Online<sup>Beta</sup></h1>
-          <p>File your BC cooperative association's annual reports and maintain your registered office addresses and director information.</p>
-
+          <h1>Start A Business And <br> Keep Records up to Date</h1>
```

Comment:
The spaces before and after the `<br>` might affect centering of the content.

Also on line 20 below.

Reply:
Interesting, i'll look at this!

Reply:
It actually trims the excess white space by default  ( i didn't know that :P ), and requires a css prop of `white-space: something` to manipulate 🤔